scripts/publish.ts

```diff
@@ -0,0 +1,148 @@
+import { readdirSync, readFileSync, writeFileSync } from 'fs'
+import { join, resolve } from 'path'
+import { chdir } from 'process'
+import { prompt, Separator } from 'inquirer'
+import * as spawn from 'await-spawn'
+import { blue, green, red } from 'colors'
+
+const readJSON = (jsonPath: string): any => {
+  return JSON.parse(
+    readFileSync(jsonPath, { encoding: 'utf-8' as BufferEncoding })
+  )
+}
+
+const writeJSON = (jsonPath: string, object: any): void => {
+  writeFileSync(jsonPath, JSON.stringify(object, null, 2) + '\n')
+}
+
+const fromEntries = (xs: [string | number | symbol, any][]) =>
```

Comment:
I think it's more clear if the argument of the function is a dictionary.
in any case, since this function is only used once and it's not clear what it does without a context, I would embed it where you use it. Btw, why not embedding the code directly within the readJson function?